configuration/pih/htmlforms/hiv/section-hiv-plan.xml

```diff
@@ -489,224 +453,104 @@
-                                            <obs conceptId="PIH:PROPHYLAXIS STARTED"
-                                                 answerConceptId="CIEL:1065" style="checkbox"
-                                                 labelCode="pihcore.initiated" answerLabel=""
-                                                 toggle="{drugname}-start-date"/>
-                                            <obs conceptId="PIH:Prophylaxis should continue"
-                                                 answerConceptId="CIEL:1065" style="checkbox"
-                                                 labelCode="pihcore.continued" answerLabel="" />
-                                            <obs conceptId="PIH:Stop Rx" style="checkbox"
-                                                 answerConceptId="CIEL:1065"
-                                                 labelCode="pihcore.stopped" answerLabel=""
-                                                 toggle="{drugname}-reason-stopped"/>
-                                        </td>
-                                        <td class="{drugname}-treat small-input">
-                                            <obs conceptId="PIH:Prescription instructions non-coded" />
-                                        </td>
-                                        <td class="{drugname}-treat">
-                                            <p class="{drugname}-start-date">
-                                                <!-- Treatment start date -->
-                                                <obs conceptId="CIEL:163526" allowFutureDates="true"
-                                                     labelCode="pihcore.startDate" />
-                                            </p>
-                                            <p class="{drugname}-reason-stopped small-input">
-                                                <obs conceptId="PIH:REASON TREATMENT STOPPED COMMENT"
-                                                     labelCode="pihcore.stopReason" />
-                                            </p>
-                                        </td>
-                                    </tr>
-                                </obsgroup>
-                            </template>
-                            <render conceptMap="CIEL:105281" drugname="Cotrimoxazole" />
-                        </repeat>
-
-                        <repeat>
-                            <template>
-                                <obsgroup groupingConceptId="PIH:10742">
-                                    <tr>
-                                        <td>
-                                            <obs conceptId="CIEL:1282" answerConceptId="{conceptMap}"
-                                                answerLabel="{drugname}"
-                                                toggle="{id: '{drugname}-treat', style: 'dim'}"/>
-                                            <p class="radio {drugname}-treat">
```

Comment:
This should stay.  We need to record INH treatment line (for reporting - I think).  I think the label should be changed to "INH treatment line" with primary/secondary OR "Isoniazid treatment line" with the same answers.  It won't be associated with the actual order, but I think it should be recorded in this section.

Otherwise, it should be fine @mseaton